### PR TITLE
Cfp update

### PR DESCRIPTION
**Pull request by xFreed0m**

### CODE REVIEW

File: content/cfp/CFP Review Board/_index.md
Comment (omercnet) [inline content/cfp/CFP Review Board/_index.md:18]: Not sure we need a dedicated page for this

Why not add this to the existing team page?

File: data/team.yml
Comment (omercnet) [inline data/team.yml:135]: We should have at least everyone's LinkedIn
Can add other socials later on -  but remove empty ones, looks bad just icons

**Review by omercnet — CHANGES_REQUESTED** [review]

File: content/cfp/CFP Review Board/_index.md
Comment (xFreed0m) [reply content/cfp/CFP Review Board/_index.md:18]: will be too crowded. I prefer to separate the core team and the cfp team

File: data/team.yml
Comment (xFreed0m) [reply data/team.yml:135]: it's just a place holder, once merge i'll work with each of them to determine which socials they want in there and will remove the empty ones

**Comment by xFreed0m** [comment]
@omercnet  why all the pictures are so big?

**Review by omercnet — APPROVED** [review]

**Pull request merged**
